wiki/content/deploy/index.md

```diff
@@ -1706,15 +1739,16 @@ If the `--tls_client_auth` option is set to `REQUEST`or `VERIFYIFGIVEN` (default
 use the option `--cacert`. For instance (for an export request):
 
 ```
-curl --cacert ./tls/ca.crt https://localhost:8080/admin/export
+curl --silent --cacert ./tls/ca.crt https://localhost:8080/admin/export
 ```
 
 If the `--tls_client_auth` option is set to  `REQUIREANY` or  `REQUIREANDVERIFY`,
 in addition to the `--cacert` option, also use the `--cert` and `--key` options.
 For instance (for an export request):
 
 ```
-curl --cacert ./tls/ca.crt --cert ./tls/node.crt --key ./tls/node.key https://localhost:8080/admin/export
+curl --silent --cacert ./tls/ca.crt --cert ./tls/client.dgraphuser.crt \
```

Comment:
Agree with Daniel. Drop the `\`

Reply:
Daniel was talking about instructional content, not a command line command, which  I accidentally had in there at the end (unless I misread).  For a command line, I folded these to multiple lines as the command requires a lot of horizontal scrolling.    Do we want the user to scroll horizontally a lot for long commands, or continue to next line vertically?

I am fine with either

Reply:
Ok. May be good to look at rest of the docs to be consistent.